processor/resourcedetectionprocessor/internal/azure/aks/metadata.yaml

```diff
@@ -10,4 +10,8 @@ resource_attributes:
   cloud.platform:
     description: The cloud.platform
     type: string
-    enabled: true
\ No newline at end of file
+    enabled: true
+  k8s.cluster.name:
+    description: The k8s.cluster.name
```

Comment:
Can we add some details in the description about how the cluster name is detected? The comments in the code are great but a user won't know to refer to them and might assume that this is set to the cluster resource name which might not be the case.

Reply:
Thank you for taking a look. I added some clarity to the metadata yaml and readme.